DomainContent/AvatarStore/Laundry/laundry.js

```diff
@@ -0,0 +1,24 @@
+//
+//  laundry.js
+//
+//  Created by Rebecca Stankus on 9/22/17.
+//  Copyright 2017 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+//  This script can be applied to an object to make it delete avatar attachments when they enter it---as a 
+//  laundry basket to get rid of attachments after trying them on.
+
+(function () {
+    var _this = this;
+
+    _this.collisionWithEntity = function (thisID, otherID, collisionInfo) {
+        var otherUserData = Entities.getEntityProperties(otherID).userData;
```

Comment:
Since you only use the user data here, can you change this to "Entities.getEntityProperties(otherID, 'userData').userData?